docs/builds/guides/migration/migration-from-ckeditor-4.md

```diff
@@ -585,7 +586,7 @@ Note: The number of options was reduced on purpose. We understood that configuri
 		</tr>
 		<tr>
 			<td><a href="/docs/ckeditor4/latest/api/CKEDITOR_config.html#cfg-smiley_columns">smiley_columns</a> <br> <a href="/docs/ckeditor4/latest/api/CKEDITOR_config.html#cfg-smiley_descriptions">smiley_descriptions</a> <br> <a href="/docs/ckeditor4/latest/api/CKEDITOR_config.html#cfg-smiley_images">smiley_images</a> <br>  <a href="/docs/ckeditor4/latest/api/CKEDITOR_config.html#cfg-smiley_path">smiley_path</a></td>
-			<td>N/A. There is no smiley plugin in CKEditor 5. However, <a href="#emoji">emoji</a> can be pasted into the rich-text editor as Unicode content.</td>
+			<td>There is no dedicated smiley plugin in CKEditor 5. However, the {@link features/text-transformation Automatic text transformation feature} may be configured to deliver emojis with shortcodes. The <a href="#emoji">emoji</a> can also be pasted into the rich-text editor as Unicode content.</td>
```

Comment:
CKE4 Smiley plugin does not provide any autocompletion, so information that CKE5 is using emojis instead seems more important than the option for using shortcodes. I would just swap both sentences so information about an option to use emoji as Unicode is first, WDYT?

Reply:
I'll make it so.